src/LodeRunner.UI/src/services/configs.test.js

```diff
@@ -40,3 +40,34 @@ describe("checkConfigInputs", () => {
     }
   });
 });
+
+describe("addDefaultsToConfig", () => {
+  it("should replace all nulls with values", () => {
+    const config = { [CONFIG.id]: "123" };
+    addDefaultsToConfig(config);
+    Object.values(config).forEach(configValue => {
+      expect(configValue).not.toBeNull();
+    })
+  })
+});
+
+describe("removeConfigDependencies", () => {
+  it("should remove max errors", () => {
+    const config = { [CONFIG.id]: "123" };
+    addDefaultsToConfig(config);
+    config[CONFIG.runLoop] = true;
+    removeConfigDependencies(config);
+    expect(config[CONFIG.duration]).not.toBeUndefined();
+    expect(config[CONFIG.randomize]).not.toBeUndefined();
```

Comment:
Instead of `.not.toBeUndefined()`, what do you think about `.toEqual(CONFIG.OPTIONS[CONFIG.randomize].default)`?